src/current/v23.1/migration-overview.md

```diff
@@ -255,7 +255,7 @@ After you [load the test data](#load-test-data), validate your queries on Cockro
 
 ##### Shadowing
 
-You can "shadow" your production workload by replicating all writes from the source database to CockroachDB. To do this, use a [change data capture (CDC)]({% link {{ page.version.version }}/cdc-queries.md %}) service. {% comment %}The CockroachDB Live Migration Service (MOLT LMS) can also write queries to both databases simultaneously.{% endcomment %} You can then [test the queries](#test-query-results-and-performance) on CockroachDB for consistency, performance, and potential issues with the migration.
+You can "shadow" your production workload by executing your source SQL traffic on CockroachDB in parallel. [MOLT LMS (Live Migration Service)]({% link {{ page.version.version }}/live-migration-service.md %}) can perform shadowing. You can then [test the queries](#test-query-results-and-performance) on CockroachDB for consistency, performance, and potential issues with the migration.
```

Comment:
Amending to "executing your source SQL statements". I'm hesitant to use "directing" or "sending" because that implies that the source traffic is being routed to the target, when my understanding is that it's simply done twice. @cotedm let me know if that's (in)correct.

Reply:
Yeah that's correct. The traffic is being sent both places in the case of shadowing.